_authors/mel-choyce.md

```diff
@@ -3,5 +3,6 @@ name: mel-choyce
 first_name: Mel
 last_name: Choyce
 full_name: Mel Choyce
-published: false
+published: true
```

Comment:
Everytime I build the site, it updates this file, so I figure it's probably supposed to be updated. I'm just gonna commit it.